src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -149,6 +151,8 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
     }),
   );
 
+  const validDuration = secondsToDuration(validDurationSeconds || 0, language);
```

Comment:
You could consider calling this one `validDurationString` to be in line with the strings declared just below, if they do the same thing?

Might also be unnecessary to declare the variable at all if `validDurationSeconds` is falsy, which is part of a check at line 314? Just thinking out loud here, not really seeing any other way to do it, since I like the approach with declaring the variable outside of the function to make it more readable, and I guess running with the default 0 won't cause any issues 👌

Reply:
Good point! By not declaring the variable at all, and moving it inside the validDurationSeconds check, we also don't need the "|| 0" :+1: